### DISCUSSION TITLE
Connection-Issues with my DTU

### DISCUSSION THREAD

**Discussion by mkroeber** [tbnobody/OpenDTU#2200] [Q&A] [answered]
Greetings,
last year i built myself a DTU with both NRF24 and CMT2300A Chip.
I did that, beacause i had an older existing HM800 running and was planning on getting a new HMT-2250-6T.
I realised that project in July this year and the inverter itself seems to be running ok.

My problem seems to be the connection to the DTU.
While my old HM800 is always online in the DTU-Webinterface (Distance DTU<-> inverter: about 10m, 2 Walls between) the HMT-2250 is often offline (Distance DTU<-> inverter: 10m free line of sight).
Here are the System Informations:

- SDK-Version | v4.4.6-dirty
- Konfigurationsversion | 0.1.27
- nRF24 Status | konfiguriert
- nRF24 Chip-Status | verbunden
- nRF24 Chip-Typ | nRF24L01+
- CMT2300A Status | verbunden
- CMT2300A Chip-Status | konfiguriert

What can i do to get a stronger connection to the HMT-2250? I already tryed playing arround with the transmission power but have no luck.
Any help would be much appreciated.

regards,
Markus

**Comment by bastianbugla85** [comment]
I have the same issue meanwhile. After mentioned solution in discussion #2192 , to increase the transmission power, I can also see now, that the connectivity is not stable. Today I don't get any response at all :( ... I don't have a solution at the moment

**Comment by JHCRausch** [comment]
Same here. I ran a DTU against a HMS-800-2T for more than a year with different releases of openDTU, without any issues. Since a couple of days, I have occasional connection issues. Already increased transmisssion power to max 20 dBm, but still. Occurence of connection issues seem to correlate with the upgrade to openDTU v24.8.5. Not sure this is the case. Any thoughts on this?

**Reply by JHCRausch** [reply]
I just saw that my DTU temperature is above 90 Celsius (HMS is 60 Celsius). Room temp is around 25 Celsius.

**Comment by mkroeber** [comment]
Yesterday i updated my OpenDTU Firmware to v24.6.10. After that i had not one connection loss till the sun went down.
Today the 2250 hasnt com online yet :-/

I bought an DTU-Lite-S so i could change some settings for the netprovider... it has no connection Problems at all... distance 20m, 3 walls between...

Could it be the firmware one the inverter? Installed is Softwarevers.V01.00.10 - that should be a fairly old one? Firmware upgrades are only possible through hoymiles support?


no clue what to do :-/

**Comment by ghost** [comment]
Hi,
Increasing the transmission power of the DTU does not have to help.
This does not improve reception on the DTU. This also does not change the transmission and reception power of the inverter.
The antenna of the CMT module may be unsuitable. 
Maybe this will help you:
#1879
https://github.com/tbnobody/OpenDTU/discussions/2106#discussioncomment-9973850

**Reply by mkroeber** [reply, marked as answer]
> Hi, Increasing the transmission power of the DTU does not have to help. This does not improve reception on the DTU. This also does not change the transmission and reception power of the inverter. The antenna of the CMT module may be unsuitable. Maybe this will help you: #1879 [#2106 (comment)](https://github.com/tbnobody/OpenDTU/discussions/2106#discussioncomment-9973850)

That helped a lot!
I got myself a new antenna... issue solved.
(https://www.amazon.de/dp/B07S24V6WQ?ref=ppx_yo2ov_dt_b_fed_asin_title)
Thanks.
